.eslintrc

```diff
@@ -72,18 +77,29 @@
             "nonwords": false
         }],
         "spaced-comment": 0,
-        "spaced-line-comment": 2,
 
         // ECMAScript 6
+        "arrow-parens": 1,
+        "arrow-spacing": [1, {
+            "before": true,
+            "after": true
+        }],
         "constructor-super": 2,
         "generator-star-spacing": [2, {
             "before": false,
             "after": true
         }],
+        "init-declarations": [2, "always"],
+        "no-class-assign": 2,
+        "no-const-assign": 2,
+        "no-invalid-this": 1,
         "no-this-before-super": 2,
         "no-var": 1,
         "object-shorthand": 0,
         "prefer-const": 1,
+        "prefer-reflect": 1,
```

Comment:
@Constellation how do you feel [this rule](http://eslint.org/docs/1.0.0/rules/prefer-reflect)? I doubt we really need to enable this.